Dockerfile.quantivly-custom

```
@@ -0,0 +1,72 @@
+FROM python:3.7-slim
+
+# these build arguments should be defined _after_ FROM, so the values are available later on
+ARG CROSSBAR_BUILD_ID
+ARG CROSSBAR_BUILD_DATE
+ARG CROSSBAR_VCS_REF
+ARG CROSSBAR_VERSION
+
+# Metadata labeling
+LABEL org.label-schema.build-date=$CROSSBAR_BUILD_DATE \
+      org.label-schema.name="Crossbar.io Starter Template" \
+      org.label-schema.description="Quickstart template for application development with Crossbar.io" \
+      org.label-schema.url="http://crossbar.io" \
+      org.label-schema.vcs-ref=$CROSSBAR_VCS_REF \
+      org.label-schema.vcs-url="https://github.com/crossbario/crossbar" \
```

I guess we can update this.
```suggestion
      org.label-schema.vcs-url="https://github.com/quantivly/crossbar" \
```